packages/babel-types/src/index.js

```diff
@@ -431,7 +424,6 @@ export function inherits(child: Object, parent: Object): Object {
   }
 
   t.inheritsComments(child, parent);
-  traverse.copyCache(parent, child);
```

Comment:
The cache is a [WeakMap](https://github.com/babel/babel/blob/dc1f40540de11045bb4714350350e012fee4f93e/packages/babel-traverse/src/cache.js#L1), so I would assume that all logic needs to work even if the cache entry is not there.
👍